### PR TITLE
Dashboard filter display

### PR DESCRIPTION
**Pull request by Alex-Aralis**
## Description
Add component that displays dashboard filters to the user.

## Proposed Changes
- Add a collapsed filter view to the dashboards screen.

## Screenshots

Single line:
image

With wrapping:
image

With no filters:
image

### CODE REVIEW

**Comment by cythrawll** [comment]
What does it look like with no global filters?

File: packages/dashboards/tests/integration/components/dashboard-filters-collapsed-filter-test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  'Integration | Component | dashboard filters collapsed filter',
+  {
+    integration: true
+  }
+);
+
+test('it renders', function(assert) {
+  this.render(hbs`{{dashboard-filters-collapsed-filter}}`);
+
+  assert.equal(
```
Comment (cythrawll) [inline packages/dashboards/tests/integration/components/dashboard-filters-collapsed-filter-test.js]: Can we get tests?

File: packages/dashboards/tests/integration/components/dashboard-filters-collapsed-filter-test.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  };
+
+  this.render(hbs`{{dashboard-filters-collapsed-filter filter=filter}}`);
+
+  assert.equal(
+    this.$()
+      .text()
+      .trim()
+      .replace(/\s+/g, ' '),
+    'Proptery equals 1, 2',
```
Comment (mattgraham1995) [inline packages/dashboards/tests/integration/components/dashboard-filters-collapsed-filter-test.js]: `Proptery` => `Property`

File: packages/dashboards/tests/integration/components/dashboard-filters-collapsed-filter-test.js [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+  };
+
+  this.render(hbs`{{dashboard-filters-collapsed-filter filter=filter}}`);
+
+  assert.equal(
+    this.$()
+      .text()
+      .trim()
+      .replace(/\s+/g, ' '),
+    'Proptery equals 1, ValueDesc (2)',
```
Comment (mattgraham1995) [inline packages/dashboards/tests/integration/components/dashboard-filters-collapsed-filter-test.js]: All the `Proptery` typos

**Pull request merged**